src/main.rs

```diff
@@ -283,6 +292,10 @@ async fn main() {
 	// Handle about pages
 	app.at("/about").get(|req| error(req, "About pages aren't added yet".to_string()).boxed());
 
+	// Instance info page
+	app.at("/instance-info").get(|r| instance_info::instance_info(r).boxed());
```

Comment:
```suggestion
	app.at("/info").get(|r| instance_info::instance_info(r).boxed());
```

I prefer `/info` since it's concise.